dashboard/app/views/home/audio_test.html.haml

```diff
@@ -0,0 +1,63 @@
+!!! 5
+%html{dir: locale_dir}
+  %head
+    = javascript_include_tag 'soundEffects'
+    :javascript
+      var enableDiagnostics = true;
+      var sounds = new Sounds(enableDiagnostics);
```

Comment:
Or maybe I should tear out the diagnostics stuff altogether?  I'm not sure how useful it is long-term.   What do you think?